server/data/vocabularies.json

```diff
@@ -0,0 +1,2246 @@
+[
+    {
+        "_id": "countries",
+        "schema": {
+            "name": {},
+            "qcode": {}
+        },
+        "type": "manageable",
+        "display_name": "Countries",
+        "unique_field": "qcode",
+        "items": [
```

Comment:
@karelpetrak told that it's better to put Belgium on top of the list. 
I'm not sure if it will change an order in the UI :)

Reply:
checked with @karelpetrak it not needed